src/postprocess/chi0.jl

```diff
@@ -204,5 +203,5 @@ function apply_χ0(ham, δV, ψ, εF, eigenvalues;
         δρ .+= ldos .* dot(ldos, δV) .* dVol ./ dos
     end
 
-    δρ .* normδV
+    from_real(basis, δρ .* normδV, assume_real=true)
```

Comment:
`;` for the kwarg. I'm not sure our interface is great here... Maybe we should make assume_real be true by default? Anything that uses RFA is real, right? Only the orbitals are complex, and they are not RFA. Of course we should also optimize by cutting the number of G vectors in half, but I'm too lazy to do that

Reply:
I'm ok with going for real by default if we keep the flag to switch off if needed.

Reply:
Why?

Reply:
Making it real-only would emphasize the difference between this (for densities/potentials) and wavefunctions